File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLRadioButtonGroupProps.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLRadioButtonGroupProps
    {
        public String CanvasID;
        public String WindowID;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public String Alignment;
        public String GroupName;
        public List<Object> Labels;
        public String SelectedID;
        public String LabelTextColor;
        public String LabelFontString;
        public String Radius;
        public List<Object> ButtonExtents;
        public String LabelTextHeight;
        public Object Tag;
        public String TabStopIndex;

        CCLRadioButtonGroupProps()
        {
            Labels = new ArrayList<Object>();
            ButtonExtents = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLRadioButtonGroupProps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLRadioButtonGroupProps
    {
        public class ButtonExtent
        {
            public int X { get; set; }
            public int Y { get; set; }
            public int Width { get; set; }
            public int Height { get; set; }
        }

        public string CanvasID { get; set; }
        public int WindowID { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public string Alignment { get; set; }
        public string GroupName { get; set; }
        public List<string> Labels { get; set; }
        public int SelectedID { get; set; }
        public string LabelTextColor { get; set; }
        public string LabelFontString { get; set; }
        public int Radius { get; set; }
        public List<ButtonExtent> ButtonExtents { get; set; }
        public int LabelTextHeight { get; set; }
        public object Tag { get; set; }
        public int TabStopIndex { get; set; }

        public CCLRadioButtonGroupProps()
        {
            Labels = new List<string>();
            ButtonExtents = new List<ButtonExtent>();
        }
    }
}